db/compaction/compaction_job.cc

```diff
@@ -2757,6 +2779,10 @@ static std::unordered_map<std::string, OptionTypeInfo>
          {offsetof(struct CompactionServiceOutputFile, marked_for_compaction),
           OptionType::kBoolean, OptionVerificationType::kNormal,
           OptionTypeFlags::kNone}},
+        {"unique_id",
```

Comment:
Maybe we should chat about how this interacts with remote compaction.

Reply:
After discussing, remote compaction does not attempt to associate the DB session id of the requesting read-write DB with files produced by remote compaction. This is perhaps not ideal for debugging but OK, and makes the remote compaction host solely responsible for ensuring no collisions in (session id, file number). So the changes here are fine as long as we are VERY careful about about the remote compactor's handling of requester's session ID.

Reply:
Good point, added comments for that.